.travis.yml

```diff
@@ -17,11 +17,12 @@ before_install:
   - conda info -a
 
 install:
-  - conda env create -n test_env --file ci/requirements-py36.yml
-  - source activate test_env
+  - conda env create -n xskillscore-dev --file ci/requirements-py36.yml
```

Comment:
Think you can do `conda env create --file ci/requirements-py36.yml` and it picks up the name